release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -422,6 +433,15 @@ In the table, features are marked with the following statuses:
 AMD and Intel 64-bit architectures (x86-64-v2) will still be supported.
 ====
 
+[id="ocp-4-10-docker-config-location-deprecation"]
+==== Default Docker configuration location deprecation
+Previously, `oc` commands that used a registry configuration would obtain credentials from the Docker configuration location, which was `~/.docker/config.json` by default. This has been deprecated and will be replaced by a Podman configuration location in a future version of {product-title}.
+
+[id="ocp-4-10-empty-file-support-deprecation"]
+==== Empty file support deprecation
+
+Support for empty files using the `--registry-config` and `--to` flags in `oc registry login` has been deprecated. Return support for `-` and `stdout` has also been deprecated in `oc registry login`. They will be removed in a future version of {product-title}.
```

Comment:
not sure if it is clear what return support means. Deprecated is passing `-` as an argument to `--registry-config` or `--to`.  Stdout is just a description what `-` means.